app/controllers/api/mixins/genealogy.rb

```diff
@@ -0,0 +1,58 @@
+module Api
+  module Mixins
+    module Genealogy
+      RELATIONSHIP_COLLECTIONS = %w[vms templates].freeze
+      VALID_EDIT_ATTRS = %w[description name child_resources parent_resource].freeze
+
+      def tag_specified(id, data)
```

Comment:
Why is this method in this mixin?  Seems unrelated to genealogy

Reply:
Oh I copy and pasted this mixin template from another file, forgot to delete that.